spaceship.zsh

```diff
@@ -926,11 +926,11 @@ spaceship_ember() {
   [[ $SPACESHIP_EMBER_SHOW == false ]] && return
 
   # Show EMBER status only for folders w/ ember-cli-build.js files
-  [[ -f ember-cli-build.js ]] || return
+  [[ -f ember-cli-build.js && -f node_modules/ember-cli/package.json ]] || return
 
   _exists ember || return
```

Comment:
As I see, this check is not useful anymore? `ember` isn't being used in this section.